pysyncthru/__init__.py

```diff
@@ -76,41 +76,41 @@ def is_unknown_state(self) -> bool:
     def model(self) -> Optional[str]:
         """Return the model name of the printer."""
         try:
-            return self.data.get("identity").get("model_name")
+            return cast(str, self.data.get("identity", {}).get("model_name"))

```

Comment:
The cast does not really help anyone but the typchecker and clutters the code unnecessarily.
In addition to cluttering the code, this makes the return type of this method (obscured) to `str` as even the possible `None` returned from `get(...)` is casted to `str`.

Maybe the dictionary can be typed to `Dict[str,str]`?

Reply:
Sometimes mypy needs a bit of hand holding. Most often fixing its errors lead towards better code though -- such as in this case _if_ the dict can be made more specific. Based on the example in the tests, the `identity` member dict is a `[str,str}, but  I don't think there's a clean way to just mark that as such given the supported Python versions, so a cast or type ignore would be needed anyway.

Reply:
Added the dict cast -- agree it's a bit better that way. But also even more verbose.